### PR TITLE
Filtering on tags [GAWB-2028]

### PR DESCRIPTION
**Pull request by aednichols**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Squash commits and merge to develop
- [x] **Submitter**: Delete branch after merge
- [x] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [x] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/main/broadfcui/page/workspaces_list.cljs [hunk 1]
```diff
@@ -17,7 +17,6 @@
    [broadfcui.net :as net]
    [broadfcui.page.workspace.create :as create]
    [broadfcui.persistence :as persistence]
-   [broadfcui.utils :as utils]
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/page/workspaces_list.cljs]: We keep this import, so you don't have to mess with imports to put a log on something.

**Review by ahaessly — APPROVED** [review]
👍

File: src/cljs/main/broadfcui/page/library/library_page.cljs [hunk 2]
```diff
@@ -282,35 +280,47 @@
...
+         (let [tags (mapv :key (:buckets aggregations))
+               ;; Don't show tags that we pulled out of persistence, but which no longer exist (workspace or tag deletion)
+               selected-tags (set/intersection (:selected-items props) (set tags))]
+           (filter/section
+            {:title title
+             :content (react/create-element
+                       [comps/TagAutocomplete
+                        {:ref "tag-autocomplete"
+                         :tags (utils/log ":tags " selected-tags)
+                         :data (utils/log ":data " (set tags))
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/page/library/library_page.cljs]: Take out logging

**Comment by MattPutnam** [comment]
👍 , just revert that one import removal

**Pull request merged**